policyengine_us/parameters/gov/local/ny/mamdani_income_tax/rate.yaml

```
@@ -0,0 +1,10 @@
+description: Zohran Mamdani's proposed this NYC income tax rate on income above a certain threshold.
```

this style for the scale param
```suggestion
description: Zohran Mamdani proposed taxing New York City residents' income above a threshold at this rate.
```